src/cata_tiles.cpp

```diff
@@ -594,40 +621,40 @@ void cata_tiles::load_ascii_set(JsonObject &entry, int offset, int size)
         id[7] = static_cast<char>(FG);
         id[8] = static_cast<char>(-1);
         tile_type &curr_tile = tile_ids[id];
-        curr_tile.fg.push_back(index_in_image + offset);
+        std::vector<int> *sprites = curr_tile.fg.add(std::vector<int>({index_in_image + offset}),1);
```

Comment:
Similar to below, if you're not checking for null anyway, make it a reference and use `operator[]`.